src/projectscene/view/clipsview/selectionviewcontroller.h

```diff
@@ -53,9 +53,9 @@ class SelectionViewController : public QObject
 private:
 
     IProjectViewStatePtr viewState() const;
-    std::vector<trackedit::TrackId> trackIdList() const;
+    trackedit::TrackIdList trackIdList() const;
 
-    std::vector<trackedit::TrackId> determinateTracks(double y1, double y2) const;
+    trackedit::TrackIdList determinateTracks(double y1, double y2) const;
 
     TimelineContext* m_context = nullptr;
 
```

Comment:
It seems to me that all `selectionActive`-related members (signal, getter, setter, state variable) are not used externally at all, as a consequence of which it should be removed to simplify and clarify the logic even more. Or is there a reason to keep it I'm missing?

Reply:
hmmm, it's actually used in TracksClipsView.qml

`property bool isNeedSelectionCursor: !selectionController.selectionActive && mainMouseArea.mouseOnTracks`